apps/Caskaydia Cove NF/install

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
+cd /tmp
 mkdir cascadia
 cd cascadia
 wget https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.1/CascadiaCode.zip || error "failed to download font"
 unzip CascadiaCode.zip || error "failed to extract zip"
-mkdir ~/.fonts
-mkdir ~/.fonts/Caskaydia
+sudo mkdir /usr/local/share/fonts
+sudo mkdir /usr/local/share/fonts/Caskaydia
```

Comment:
use `sudo mkdir -p /usr/local/share/fonts/Caskaydia` instead of both of these lines. it will create subfolders as necessary